packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -197,6 +197,7 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
 
   static propTypes = {
     ...commonPropTypes.createCommon({
+      accessibility: false,
```

Comment:
Are we sure which component should not have the accessibility property?

Reply:
Yep, it's not implemented yet #875